README.md

```diff
@@ -0,0 +1,28 @@
+[![Build Status](https://travis-ci.org/RIOT-OS/applications.svg?branch=master)](https://travis-ci.org/RIOT-OS/applications)
+
+# RIOT Applications
+
+This repository provides more applications for the [RIOT operating system][riot-repo].
+Some of them are just useful tools for development work, others showcase
+software provided with RIOT beyond the simple examples within the RIOT codebase.
+
```

Comment:
Some of them are useful tools for development work, others showcase more extensive implementations of features in RIOT compared to the simple examples included in the [RIOT main codebase][riot-repo/examples].